migrations/founders.json

```diff
@@ -0,0 +1,83 @@
+{
+  "founders": {
+    "ganache": [
+      {
+        "address": "0x8d4408014d165ec69d8cc9f091d8f4578ac5564f376f21887e98a6d33a6e3549",
```

Comment:
Why these adrresses are longer that kovan addresses ?
You can use the same address for kovan and ganache..no ?

Reply:
these are private keys that ganache seems to like.  they correspond to the accounts that we tell ganache to create when we fire it up.

Reply:
The kovan addresses correspond to accounts on kovan that won't exist on ganache.

Reply:
- please replace private key with address .
- if it is not possible  change : "address" tp "private key" or at least comment that.

Reply:
yeah these should all be public keys, on ganache too. the private keys won't work

Reply:
changed to addresses